assets/javascripts/discourse/components/reviewable-post-voting-comment.js

```diff
@@ -0,0 +1,15 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { inject as service } from "@ember/service";
+
+export default class ReviewableChatMessage extends Component {
```

Comment:
Is this supposed to be `ReviewableChatMessage`? 🚨

Reply:
Yup I swear I changed this like 5 times but always got back jajaja